pkg/commands/process/settings/rules/ruby/lang/weak_encryption.yml

```diff
@@ -34,6 +34,11 @@ patterns:
           - export
           - to_pem
           - to_s
+  - pattern: |
+      OpenSSL::PKey::RSA.new($<LENGTH>)
```

Comment:
Do we want DSA here too? 

https://ruby-doc.org/stdlib-3.1.1/libdoc/openssl/rdoc/OpenSSL/PKey/DSA.html